app/addons/components/assets/less/layouts.less

```diff
@@ -19,3 +19,11 @@
   flex-direction: row;
   height: 100%;
 }
+
+//yes !important!!! We have to overridde some styles I can't override any other way for now
```

Comment:
wait a second... this css overrides it globally everywhere.

why do we need to override then - instead of replacing the old CSS?